ui-particles-angular/projects/ui-particles-angular/src/lib/gio-form-json-schema/gio-form-json-schema.stories.component.html

```diff
@@ -15,11 +15,36 @@
     limitations under the License.
 
 -->
-<mat-card>
-  <mat-card-title>{{ title }}</mat-card-title>
-  <br />
-  <form [formGroup]="form" (ngSubmit)="onSubmit()">
-    <formly-form [model]="model" [fields]="fields" [options]="options" [form]="form"></formly-form>
-    <button type="submit">Submit</button>
-  </form>
-</mat-card>
+<div class="json-schema-storie">
```

Comment:
```suggestion
<div class="json-schema-story">
```

We must choose between singular `story` or plural `stories` 😁

Reply:
Choosing is giving up 🙈